### PR TITLE
Snake smash

### PR DESCRIPTION
**Pull request by jamesaimonetti**

### CODE REVIEW

File: applications/crossbar/doc/api_authentication.md [hunk 1]
```diff
@@ -32,9 +32,6 @@ Key | Description | Type | Default | Required
 --- | ----------- | ---- | ------- | --------
 `api_key` | The Accounts API key | `string(64)` |   | `true`
 
-
-
-
```
Comment (fenollp) [inline applications/crossbar/doc/api_authentication.md]: these were here for scripts/generate-doc-schemas.sh
I don't remember the exact amount of newlines needed so not sure if this breaks that. Can you check?

File: applications/crossbar/doc/resource_selectors.md [hunk 2]
```diff
@@ -17,10 +17,7 @@ Key | Description | Type | Default | Required
 `stop_time` | Stop time (gregorian seconds) | `integer` |   | `false`
 `value` | Extra selector data | `string` |   | `false`
 
-
-
-
```
Comment (fenollp) [inline applications/crossbar/doc/resource_selectors.md]: same here

File: applications/crossbar/doc/temporal_rules.md [hunk 3]
```diff
@@ -21,8 +21,7 @@ Key | Description | Type | Default | Required
 
 
 
-
```
Comment (fenollp) [inline applications/crossbar/doc/temporal_rules.md]: ditto

File: applications/crossbar/doc/temporal_rules_sets.md [hunk 4]
```diff
@@ -14,9 +14,6 @@ Key | Description | Type | Default | Required
 `temporal_rules` | Temporal Rules | `array(string)` |   | `false`
 `temporal_rules.[]` |   | `string` |   | `false`
 
-
-
-
```
Comment (fenollp) [inline applications/crossbar/doc/temporal_rules_sets.md]: ditto

**Comment by jamesaimonetti** [comment]
@fenollp addressed and updated

**Pull request merged**